02_run_all_in_one.sh

```diff
@@ -45,10 +35,17 @@ parameter_defaults:
   StandaloneHomeDir: /home/$USER
   StandaloneLocalMtu: 1500
   # Needed if running in a VM
-  StandaloneExtraConfig:
-    nova::compute::libvirt::services::libvirt_virt_type: qemu
-    nova::compute::libvirt::libvirt_virt_type: qemu
+  NovaComputeLibvirtType: qemu
 EOF_CAT
 
-sudo openstack tripleo deploy  --templates $SCRIPTDIR/tripleo-heat-templates --local-ip=$LOCAL_IP/$CIDR   -e $SCRIPTDIR/tripleo-heat-templates/environments/standalone.yaml -r $SCRIPTDIR/standalone.yaml -e $SCRIPTDIR/standalone_parameters.yaml --output-dir $SCRIPTDIR/standalone --standalone -e $SCRIPTDIR/tripleo-heat-templates/environments/enable-designate.yaml
+sudo openstack tripleo deploy \
+    --templates $SCRIPTDIR/tripleo-heat-templates \
+    --local-ip=$LOCAL_IP/$CIDR \
+    -e $SCRIPTDIR/containers-prepare-parameters.yaml \
+    -e $SCRIPTDIR/tripleo-heat-templates/environments/standalone/standalone-tripleo.yaml \
+    -r $SCRIPTDIR/standalone.yaml \
```

Comment:
we actually ship this file in THT now, so not sure if you still need this in this repo